build.gradle

```diff
@@ -255,3 +255,24 @@ def deployer = uploadArchives.repositories.mavenDeployer
     }.each { dep -> dep.optional = true }
 }
 
+
+apply plugin: 'findbugs'
+//findbugs
+findbugs {
+    toolVersion = "3.0.1"
+    //report only high severity bugs for now
+    reportLevel = "high"
+    //the code base is pretty small so use max effort
+    effort = "max"
+    //we don't want to run findbugs on the test code yet
+    sourceSets = [sourceSets.main]
+    //exclude a couple of known bugs until we get the chance to fix them
+    excludeFilter = file("findbugs_excludes.xml")
+}
+
+tasks.withType(FindBugs) {
+    reports {
+        xml.enabled = false
```

Comment:
There a j[enkins plug-in](https://wiki.jenkins-ci.org/display/JENKINS/FindBugs+Plugin) to pick up the findbugs report, its not installed yet, but when we set up the build we can do that. So I think we should turn on the xml output